styles.css

```diff
@@ -6,4 +6,19 @@ h1 {
 main {
   display: flex;
   justify-content: center;
+  align-items: center;
+  flex-direction: column;
 }
+
+.div-p {
+  font-size: 14px;
+}
+
+ul.ul-test {
+  display: block;
+  list-style: none;
+}
```

Comment:
@NelsonAraujo93  if you only have one <ul> element is not necesary create a class to the element, you must create clases when you have multiple elements with the same look and feel, but this is not the case.